iwyu_include_picker.cc

```diff
@@ -1303,15 +1303,15 @@ vector<string> IncludePicker::GetCandidateHeadersForFilepath(
 vector<string> IncludePicker::GetCandidateHeadersForFilepathIncludedFrom(
     const string& included_filepath, const string& including_filepath) const {
   vector<string> retval;
-  const string quoted_includer = ConvertToQuotedInclude(including_filepath);
-  const string quoted_includee = ConvertToQuotedInclude(included_filepath);
+  const string quoted_includer = ConvertToQuotedInclude(including_filepath, MakeAbsolutePath(GetParentPath(including_filepath)));
```

Comment:
This seems logically incorrect. We don't know who includes `including_filepath` and assuming it includes itself is far-fetched. Also keeping old 

`const string quoted_includer = ConvertToQuotedInclude(including_filepath);`

doesn't break anything (probably it indicates some tests are missing).